src/Observables.jl

```diff
@@ -35,6 +35,12 @@ let count=0
     end
 end
 
+function Base.show(io::IO, x::Observable{T}) where T
+    print(io, "Observable{$T}(...) with $(length(x.listeners)) listeners")
```

Comment:
I'm a bit confused, why not show the value of the observable?

Reply:
See the commit message:
> don't print val since there is no way to print a one-line representation of an arbitrary type

The alternative would be
```
julia> Observable(df)
Observable{DataFrame} with 0 listeners. Value:
30000000×1 DataFrame
│ Row      │ a        │
│          │ Float64  │
├──────────┼──────────┤
│ 1        │ 0.623558 │
│ 2        │ 0.65565  │
│ 3        │ 0.373987 │
│ 4        │ 0.775368 │
│ 5        │ 0.523611 │
│ 6        │ 0.956819 │
│ 7        │ 0.804922 │
│ 8        │ 0.765594 │
│ 9        │ 0.394248 │
│ 10       │ 0.982146 │
⋮
│ 29999990 │ 0.676106 │
│ 29999991 │ 0.890396 │
│ 29999992 │ 0.230561 │
│ 29999993 │ 0.969732 │
│ 29999994 │ 0.53987  │
│ 29999995 │ 0.638849 │
│ 29999996 │ 0.765199 │
│ 29999997 │ 0.419617 │
│ 29999998 │ 0.764144 │
│ 29999999 │ 0.985964 │
│ 30000000 │ 0.403513 │
```
but I'm not sure I like that.

Reply:
I was comparing with similar things (`Ref` and `Array{T, 0}`):

```julia
julia> Ref(df)
Base.RefValue{DataFrame}(10×1 DataFrame
│ Row │ x     │
│     │ Int64 │
├─────┼───────┤
│ 1   │ 1     │
│ 2   │ 2     │
│ 3   │ 3     │
│ 4   │ 4     │
│ 5   │ 5     │
│ 6   │ 6     │
│ 7   │ 7     │
│ 8   │ 8     │
│ 9   │ 9     │
│ 10  │ 10    │)
```

```julia
julia> fill(df)
0-dimensional Array{DataFrame,0}:
10×1 DataFrame
│ Row │ x     │
│     │ Int64 │
├─────┼───────┤
│ 1   │ 1     │
│ 2   │ 2     │
│ 3   │ 3     │
│ 4   │ 4     │
│ 5   │ 5     │
│ 6   │ 6     │
│ 7   │ 7     │
│ 8   │ 8     │
│ 9   │ 9     │
│ 10  │ 10    │
```

And I agree that `Ref` looks a bit weird but `Array{T,0}` (pretty much the same as your alternative suggestion) does a nice job.

Reply:
Alright, I've changed the show method again :P

Reply:
Cool, I'll wait a little bit longer to see if somebody has strong opinions on how to display the info and then merge.

Reply:
Would probably be a good idea to squash when merging.